repl/evaluator_test.go

```diff
@@ -16,8 +16,18 @@ package main
 
 import (
 	"testing"
+
+	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
 )
 
+func MustParseType(t string) *exprpb.Type {
```

Comment:
No need to export this function since it's local to the test.

The preferred way to introduce helpers or asserts into tests is to make a test helper:

```go
func mustParseType(t testing.TB, typeName string) *exprpb.Type {
  t.Helper()
  ty, err := ParseType(typeName)
  if err != nil {
    t.Fatalf("ParseType(%s) failed: %v", typeName, err)
  }
  return ty
}
```

Reply:
Done